packages/stylelint-plugin/test.scss

```diff
@@ -1,30 +0,0 @@
-@import "~@kaizen/design-tokens/sass/spacing";
```

Comment:
Ah, glad to see this removed - it's been throwing false positives for me in the prepush hook.